unit-test/fm_app_tests.c

```diff
@@ -242,9 +243,11 @@ void Test_FM_AppInit_EVSRegisterNotSuccess(void)
 
     /* Assert */
     UtAssert_STUB_COUNT(CFE_EVS_Register, 1);
-    UtAssert_STUB_COUNT(CFE_EVS_SendEvent, 1);
-    UtAssert_INT32_EQ(context_CFE_EVS_SendEvent[0].EventID, FM_STARTUP_EVENTS_ERR_EID);
-    UtAssert_INT32_EQ(context_CFE_EVS_SendEvent[0].EventType, CFE_EVS_EventType_ERROR);
+    UtAssert_STUB_COUNT(CFE_ES_WriteToSysLog, 1);
+
+    call_count_CFE_EVS_SendEvent = UT_GetStubCount(UT_KEY(CFE_EVS_SendEvent));
+    UtAssert_True(call_count_CFE_EVS_SendEvent == 0, "CFE_EVS_SendEvent was called %u time(s), expected 0",
+                  call_count_CFE_EVS_SendEvent);
```

Comment:
I would recommend taking out these two assertions, no real need to assert that a function was not called.  Also no need to add the global variable either.

Reply:
No worries @jphickey 
Removed those two.